definitions/tools/add_strelka_gt.cwl

```diff
@@ -88,6 +109,12 @@ requirements:
             sub parse_gt {
                 my ($gt_str, $ids) = @_;
                 my @gt_ids = map{$ids->{$_}}(split //, $gt_str);
+                my $i;
+                foreach $i (@gt_ids) {
+                    if (!defined($i)) {
```

Comment:
This is completely fine; just noting that this is equivalent to:
```perl
unless (defined $i) {
```
in case you like having fewer operators and/or parentheses.